test/system/reports_test.rb

```diff
@@ -3,45 +3,57 @@
 require 'application_system_test_case'
 
 class ReportsTest < ApplicationSystemTestCase
+  include Warden::Test::Helpers
+
   setup do
-    @report = reports(:one)
+    @user = FactoryBot.create(:user, email: 'alice@example.com')
+    login_as(@user)
   end
 
   test 'visiting the index' do
     visit reports_url
-    assert_selector 'h1', text: 'Reports'
+    assert_selector 'h1', text: '日報の一覧'
   end
 
-  test 'should create report' do
-    visit reports_url
-    click_on 'New report'
-
-    fill_in 'Content', with: @report.content
-    fill_in 'Title', with: @report.title
-    fill_in 'User', with: @report.user_id
-    click_on 'Create Report'
+  test 'create a new report' do
+    visit new_report_url
+    fill_in 'タイトル', with: 'First report'
+    fill_in '内容', with: 'はじめまして'
+    click_button '登録する'
 
-    assert_text 'Report was successfully created'
-    click_on 'Back'
+    assert_text '日報が作成されました'
+    assert_text 'First report'
+    assert_text 'はじめまして'
+    assert_text "作成者: #{@user.name}"
   end
 
-  test 'should update Report' do
-    visit report_url(@report)
-    click_on 'Edit this report', match: :first
+  test 'update a report' do
+    report = FactoryBot.create(:report, user: @user)
 
-    fill_in 'Content', with: @report.content
-    fill_in 'Title', with: @report.title
-    fill_in 'User', with: @report.user_id
-    click_on 'Update Report'
+    visit edit_report_url(report.id)
+    fill_in 'タイトル', with: 'アップデートしてみました'
+    fill_in '内容', with: 'アップデートをした文章です。'
+    click_button '更新する'
 
-    assert_text 'Report was successfully updated'
-    click_on 'Back'
+    assert_text '日報が更新されました'
+    assert_text 'アップデートしてみました'
+    assert_text 'アップデートをした文章です。'
+    assert_text "作成者: #{@user.name}"
   end
 
-  test 'should destroy Report' do
-    visit report_url(@report)
-    click_on 'Destroy this report', match: :first
+  test 'destory a report' do
+    report = FactoryBot.create(:report, user: @user, title: '削除予定です', content: '削除をする日報です。')
+
+    visit report_url(report.id)
+
+    assert_text '削除予定です'
+    assert_text '削除をする日報です。'
```

Comment:
```suggestion
    assert_text '削除予定です'
```

タイトルが消えたことだけを検証すれば十分かなーと思います。

Reply:
修正いたしました。確かに日報削除のシステムテストにはタイトルだけで十分ですね。